### PR TITLE
update metadata and fork

### PR DESCRIPTION
**Pull request by gionn**

### CODE REVIEW

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [BaseTest.java#L14](https://github.com/ClouDesire/kong-java-client/blob/bb8f6055b7eee69f4dcb8067829a858c8bb6a189/src/test/java/com/github/vaibhavsinha/kong/BaseTest.java#L14): Add some tests to this class. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2187)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'

File: src/main/java/com/github/vaibhavsinha/kong/impl/helper/RetrofitServiceCreator.java [hunk 1]
```diff
@@ -1,14 +1,16 @@
 package com.github.vaibhavsinha.kong.impl.helper;
 
 //import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
```
Comment (AndreaVecchi) [inline src/main/java/com/github/vaibhavsinha/kong/impl/helper/RetrofitServiceCreator.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/main/java/com/github/vaibhavsinha/kong/impl/helper/RetrofitServiceCreator.java [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+
 import com.github.vaibhavsinha.kong.utils.HttpsUtil;
 import okhttp3.OkHttpClient;
 import retrofit2.Retrofit;
-//import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
-//import retrofit2.converter.gson.GsonConverterFactory;
 
 import java.lang.reflect.Proxy;
 
+//import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
```
Comment (AndreaVecchi) [inline src/main/java/com/github/vaibhavsinha/kong/impl/helper/RetrofitServiceCreator.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 51 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 3 critical
* ![MAJOR][MAJOR] 13 major
* ![MINOR][MINOR] 32 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [HttpsUtil.java#L88](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L88): Do not unconditionally return true in this method. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3510)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L195](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L195): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![MAJOR][MAJOR] [PluginService.java#L26](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java#L26): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [RetrofitApiService.java#L32](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java#L32): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L85](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L85): Remove this unused private "UnSafeHostnameVerifier" class. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3985)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L109](https://github.com/ClouDesire/kong-java-client/blob/99c11f2586efabba506da007ca52409e226db6bc/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L109): Return an empty array instead of null. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1168)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 51 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 3 critical
* ![MAJOR][MAJOR] 13 major
* ![MINOR][MINOR] 32 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [HttpsUtil.java#L88](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L88): Do not unconditionally return true in this method. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3510)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L195](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L195): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![MAJOR][MAJOR] [PluginService.java#L26](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java#L26): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [RetrofitApiService.java#L32](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java#L32): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L85](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L85): Remove this unused private "UnSafeHostnameVerifier" class. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3985)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L109](https://github.com/ClouDesire/kong-java-client/blob/2d9fa0a0308088257582945bdcf9effe3c5f6942/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L109): Return an empty array instead of null. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1168)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 51 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 3 critical
* ![MAJOR][MAJOR] 13 major
* ![MINOR][MINOR] 32 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [HttpsUtil.java#L88](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L88): Do not unconditionally return true in this method. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3510)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L195](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L195): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![MAJOR][MAJOR] [PluginService.java#L26](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java#L26): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [RetrofitApiService.java#L32](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java#L32): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L85](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L85): Remove this unused private "UnSafeHostnameVerifier" class. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3985)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L109](https://github.com/ClouDesire/kong-java-client/blob/f7486d5c99d3d24b7f927cd9570d683a5f0c0e75/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L109): Return an empty array instead of null. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1168)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: pom.xml [hunk 3]
```diff
@@ -23,25 +23,23 @@
 
 	<developers>
 		<developer>
-			<name>Vaibhav Sinha</name>
```
Comment (malteo) [inline pom.xml]: keep original contributor

**Review by malteo — CHANGES_REQUESTED** [review]

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 48 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 3 critical
* ![MAJOR][MAJOR] 11 major
* ![MINOR][MINOR] 31 minor
* ![INFO][INFO] 2 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [HttpsUtil.java#L88](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L88): Do not unconditionally return true in this method. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3510)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [HttpsUtil.java#L195](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L195): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1186)
1. ![MAJOR][MAJOR] [PluginService.java#L26](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java#L26): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [RetrofitApiService.java#L32](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java#L32): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L85](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L85): Remove this unused private "UnSafeHostnameVerifier" class. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS3985)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L94](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L94): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L98](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L98): Change this method so it throws exceptions. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS4424)
1. ![MAJOR][MAJOR] [HttpsUtil.java#L109](https://github.com/ClouDesire/kong-java-client/blob/e31817bf3c30b1bf2eb2a86b6490c2ac99c797da/src/main/java/com/github/vaibhavsinha/kong/utils/HttpsUtil.java#L109): Return an empty array instead of null. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1168)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 12 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 8 minor
* ![INFO][INFO] 2 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [PluginService.java#L26](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java#L26): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MAJOR][MAJOR] [RetrofitApiService.java#L32](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java#L32): Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
1. ![MINOR][MINOR] [KongClient.java#L68](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L68): Remove this use of "CertificateService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L69](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L69): Remove this use of "SniService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L70](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L70): Remove this use of "UpstreamService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L71](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L71): Remove this use of "TargetService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L118](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L118): Remove this use of "CertificateService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L119](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L119): Remove this use of "SniService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L120](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L120): Remove this use of "UpstreamService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L121](https://github.com/ClouDesire/kong-java-client/blob/2112d39136be3512b311deb208a4753d916568cb/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L121): Remove this use of "TargetService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java [hunk 4]
```diff
@@ -23,6 +22,9 @@
 
     Plugin updatePlugin(String nameOrId, Plugin request);
 
+    /**
+     * @deprecated use {@link #addPlugin(Plugin)} or {@link #updatePlugin(String, Plugin)}
+     */
     @Deprecated
     Plugin createOrUpdatePlugin(Plugin request);
```
Comment (AndreaVecchi) [inline src/main/java/com/github/vaibhavsinha/kong/api/admin/PluginService.java:29]: INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

File: src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java [hunk 5]
```diff
@@ -20,6 +27,9 @@
     @PATCH("apis/{id}")
     Call<Api> updateApi(@Path("id") String nameOrId, @Body Api request);
 
+    /**
+     * @deprecated use {@link #createApi(Api)} or {@link #updateApi(String, Api)}
+     */
     @Deprecated
     @PUT("apis/")
     Call<Api> createOrUpdateApi(@Body Api request);
```
Comment (AndreaVecchi) [inline src/main/java/com/github/vaibhavsinha/kong/internal/admin/RetrofitApiService.java:35]: INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 10 issues
* ![MINOR][MINOR] 8 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [KongClient.java#L68](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L68): Remove this use of "CertificateService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L69](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L69): Remove this use of "SniService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L70](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L70): Remove this use of "UpstreamService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L71](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L71): Remove this use of "TargetService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L118](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L118): Remove this use of "CertificateService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L119](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L119): Remove this use of "SniService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L120](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L120): Remove this use of "UpstreamService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [KongClient.java#L121](https://github.com/ClouDesire/kong-java-client/blob/c6f79f21fe0ad8e3047c6d32d5965d3e9efdc4ee/src/main/java/com/github/vaibhavsinha/kong/impl/KongClient.java#L121): Remove this use of "TargetService"; it is deprecated. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by malteo — APPROVED** [review]

**Pull request merged**